src/modality-specific-files/magnetic-resonance-imaging-data.md

```diff
@@ -356,6 +356,32 @@ recommended metadata fields, and the application specific entity or
 metadata requirement levels of [file collections](../appendices/file-collections.md) that can generate
 them, visit the [qMRI appendix](../appendices/qmri.md).
 
+### Deidentification information
+
+Describes the mechanism or method used to modify or remove metadata
+and/or pixel data to protect the patient or participant's identity.
+
+<!-- This block generates a metadata table.
+These tables are defined in
+  src/schema/rules/sidecars
+The definitions of the fields specified in these tables may be found in
+  src/schema/objects/metadata.yaml
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_sidecar_table("mri.DeidentificationMethod") }}
+
+Each object in the `DeidentificationMethodCodeSequence` array includes the following RECOMMENDED keys:
+
+<!-- This block generates a table describing subfields within a metadata field.
+The definitions of these fields can be found in
+  src/schema/objects/metadata.yaml
+and a guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_subobject_table("metadata.DeidentificationMethodCodeSequence.items") }}
+
+
```

Comment:
I think we should add this after `### Tissue description` (L165-L176), since this is metadata that could in principle apply to any image, not just anatomical images. While you're at it, would you mind removing one of the hash marks (`#`) from "Tissue description", since it's incorrectly being interpreted as a subsection of "Echo-Planar Imaging and B0 mapping":

![image](https://github.com/bids-standard/bids-specification/assets/83442/6955ff23-5475-4e29-81f1-e49ef8ba590e)